sales.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Sales Projections</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <section>
+      <ul id="firstAndPike-projections">
```

Comment:
Can you think of a way that you could generate these uls in addition to the lis instead of having to hard-code them into your html from the beginning?